exercise-15/src/com/nayema/Actor.java

```diff
@@ -0,0 +1,15 @@
+package com.nayema;
+
+public class Actor {
+    private static String[] name = {"Arnold Schwarzenegger", "Will Smith", "Anglenia Jolie", "Brad Pitt"};
+
+    private Actor() {
+    }
+
+    public static void randomActorName() {
```

Comment:
Its fine to use a helper method that generates a random name. However this is not a factory method. It does not create an instantiated `Actor`.